src/CabanaPD_Particles.hpp

```diff
@@ -583,6 +594,8 @@ class Particles<MemorySpace, LPS, Dimension>
         log( std::cout, "No particle output enabled." );
 #endif
 #endif
+
+        _timer.stop();
```

Comment:
@streeve Do we need below this the following?
auto time() { return _timer.time(); };
auto timeInit() { return 0.0; };

Reply:
It re-uses the base class